notebooks/BikeShareExample.ipynb

```diff
@@ -0,0 +1,815 @@
+    "    print(predictorStatus)\n",
+    "    if predictorStatus != 'ACTIVE' and predictorStatus != 'FAILED':\n",
+    "        sleep(30)\n",
+    "    else:\n",
+    "        break"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get Error Metrics\n",
+    " This provides metrics on the accuracy of the models that was  trained with the CreatePredictor operation.\n",
+    " Use metrics to see how well the model did and to decide whether to use the predictor to generate forecasts "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "forecastquery.get_accuracy_metrics(PredictorName=predictorName)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Deploy Predictor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "forecast.deploy_predictor(PredictorName=predictorName)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "deployedPredictorsResponse=forecast.list_deployed_predictors()\n",
+    "print(deployedPredictorsResponse)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Please note that the following cell can also take 10 minutes or more to be fully operational. There's no output here, but that is fine as long as the * is there."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "while True:\n",
+    "    deployedPredictorStatus = forecast.describe_deployed_predictor(PredictorName=predictorName)['Status']\n",
+    "    print(deployedPredictorStatus)\n",
+    "    if deployedPredictorStatus != 'ACTIVE' and deployedPredictorStatus != 'FAILED':\n",
+    "        sleep(30)\n",
+    "    else:\n",
+    "        break\n",
+    "print(deployedPredictorStatus)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get Forecast"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When the solution is deployed and forecast results are ready, you can view them. \n",
+    "The result set shows the predictions and confidence on the p10,p50 rates. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    " forecastResponse = forecastquery.get_forecast(\n",
+    "    PredictorName=predictorName,\n",
+    "    Interval=\"hour\",\n",
+    "   Filters={\"item_id\":\"bike_12\"}\n",
+    ")\n",
+    "print(forecastResponse)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Generate Plot \n",
+    "print(\"Forcast for P90\") \n",
```

Comment:
Should be `Forecast for mean prediction`?

Reply:
I have fixed the print statement